pom.xml

```diff
@@ -331,12 +331,14 @@
             <goals>
               <goal>checkstyle</goal>
             </goals>
-            <phase>verify</phase>
+            <phase>validate</phase>
```

Comment:
not sure whether to keep here or not, but waiting for unit test suite for linting is not great